<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerSettings</class>
 <widget class="QWidget" name="ServerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Host:</string>
     </property>
     <property name="buddy">
      <cstring>hostLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="LineEdit" name="hostLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Port:</string>
     </property>
     <property name="buddy">
      <cstring>portSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="portSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>66000</number>
     </property>
     <property name="value">
      <number>6600</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3b">
     <property name="text">
      <string>Password:</string>
     </property>
     <property name="buddy">
      <cstring>passwordLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="LineEdit" name="passwordLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Music Folder:</string>
     </property>
     <property name="buddy">
      <cstring>mpdDir</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="DirRequester" name="mpdDir"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DirRequester</class>
   <extends>QLineEdit</extends>
   <header>dirrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
